sdk/core/azure-core/azure/core/_pipeline_client_async.py

```diff
@@ -101,16 +112,34 @@ def _build_pipeline(self, config, **kwargs): # pylint: disable=no-self-use
         policies = kwargs.get('policies')
 
         if policies is None:  # [] is a valid policy list
+            per_call_policies = kwargs.get('per_call_policies')
+            per_retry_policies = kwargs.get('per_retry_policies')
             policies = [
                 RequestIdPolicy(**kwargs),
                 config.headers_policy,
                 config.user_agent_policy,
                 config.proxy_policy,
-                ContentDecodePolicy(**kwargs),
+                ContentDecodePolicy(**kwargs)
+            ]
+            if per_call_policies:
+                if isinstance(per_call_policies, (AsyncHTTPPolicy, SansIOHTTPPolicy)):
```

Comment:
Isn't per_call_policies and Iterable?

Reply:
Never mind, missed the single-or-list. I would have switched the test around and checked if it is an Iterable...